src/core/geometry/coordinate/Group.hpp

```diff
@@ -17,7 +17,7 @@ Group<C>::Group(const std::vector<Math::CVecI3>& pos)
 }
 
 template<typename C> template<typename VEC>
-typename IdentifiableUnique<C>::iterator 
+typename Class::Group::GroupIdentifiableUnique<C>::iterator
```

Comment:
I think i would remove the Group namespace to avoid this redundancy in naming. I think it kind of made sense when we had several kinds of groups. All have been unified now so maybe we can avoid that.